templates/gapic/shared/_client.erb

```diff
@@ -1,2 +1,194 @@
+<% @requires = capture do %>
+require "json"
+require "pathname"
+
+require "google/gax"
+require "google/gax/operation"
+require "google/longrunning/operations_client"
+
+require "google/cloud/<%= service.name.underscore %>/v1/cloud_<%= service.name.underscore %>_pb"
```

Comment:
The version appears in a few places but needs to be generated, but we can get that with the other items like `SERVICE_ADDRESS`, `ALL_SCOPES`, and the package name once we settle on the bigger issues.

Reply:
Same goes for all of the `google/cloud`, `cloud_`, `Google::Cloud`, etc. references.